tensorrt_llm/_torch/modules/attention.py

```diff
@@ -79,12 +81,15 @@ def __init__(
         # tensor parallel
         config = config or ModelConfig()
         tp_size = config.mapping.tp_size
-        tp_rank = config.mapping.tp_rank
+        pp_size = config.mapping.pp_size
+        rank = config.mapping.rank
         gpus_per_node = config.mapping.gpus_per_node
         if config.mapping.enable_attention_dp:
             tp_size = 1
-            tp_rank = 0
+            pp_size = 1
```

Comment:
Why is `pp_size` set to 1 here? You can have pp > 1 with attn_dp.

Reply:
Done, revert this change.